src/library-authoring/studio-header/index.scss

```diff
@@ -57,26 +57,39 @@
   nav {
     .dropdown {
       .btn {
-        font-size: 1.6rem;
+        font-size: 1rem;
         font-weight: 600;
         background-color: white;
         color: $gray-d1;
         border-width: 1px;
-        line-height: 2.368rem;
+        line-height: 1.475rem;
+        padding: 0.234rem 0.469rem;
+        border-radius: ($baseline*0.1);
+        border-color: #f8f9fa;
 
         &:hover {
           color: $uxpl-blue-hover-active;
         }
+
+        &:focus:before {
+          border-radius: 2px;
```

Comment:
above you used a `border-radius: ($baseline*0.1);` instead here - `px`
please bring to a unified style

Reply:
it uses without calculation in lots of cases in the app